doc/user_guide/metadata_structure.rst

```diff
@@ -70,7 +70,6 @@ in the following sections of this chapter.
         │   │   └── parameters_estimation_method
         │   └── variance
         ├── binned
-        ├── record_by
         ├── signal_type
         └── signal_origin
```

Comment:
Not directly related to this PR, but shouldn't `signal_origin` be also gone?
Also later (line 293), a deprecation warning could be added to `signal_origin` in the explanations